### ISSUE TITLE
Auto exiting from Google colab while training model

### ISSUE THREAD

**Issue by knightperfectionist** [nedap/deidentify#50]
I am training a model on the Spanish dataset. Also, using Spanish multi-task CNN trained  es_core_news_sm model. I refactored the code wherever it is required. While training model on Google Colab, cell automatically stops executing after a few ietrations of the very 1st epoch.  I tried changing parameters like number of epochs, batch size, learning rate and size of dataset as well but I'm getting the same results.

**Comment by jantrienes** [comment]
Thanks for reaching out about training the models on Google Colab. 

Can you please share the following information?
- Code
- Error message
- `deidentify.__version__`, `torch.__version__` and `flair.__version__`?

That will help to debug this issue.

**Comment by knightperfectionist** [comment]
Hi

Versions -
deidetify - 0.7.0
Torch - 1.7.0+cu101
flair - 0.4.3

I am not getting any error message as such, just auto terminating from cell.
I've added the attachment.

Thank you.

Pushkar Kulkarni
Student

**Comment by knightperfectionist** [comment]
Hi

I am using the same code of de identify and trying to execute.
